### ISSUE TITLE
Bug: Auf Prod funktionieren Standardantworten in Meldungen nicht

### ISSUE THREAD

**Issue by Naadjjaa** [decidim-swiss/decidim.swiss#109]
Bisherige Erkenntnisse:

PROD: 
- Als Schätzer werden die angewählten Standardantworten nicht ins Feld übernommen (hatten wir schon mal)

INT: 
- Als Schätzer werden die Standardantworten nicht ins Feld übernommen.
- Als Moderator funktioniert beides

**Comment by carlobeltrame** [comment]
Zu den Standardantworten habe ich ein Bug-Issue bei pokecode eröffnet: https://github.com/openpoke/decidim-module-reporting-proposals/issues/78
Das ist etwas was effektiv nur sie fixen können.

**Comment by carlobeltrame** [comment]
Dass die Beantwortung von Meldungen nicht funktioniert kann ich lokal nicht nachstellen. Kannst du mir genauere Informationen geben, in welcher Meldung du genau welche Antwort erfassen wolltest? Es könnte an den genauen Eingaben liegen die du machst, oder allenfalls an der WAF, aber um das zu klären muss ich es erst reproduzieren können, und auf PROD schicke ich nur ungern Test-Antworten ab.

**Comment by Naadjjaa** [comment]
[https://github.com/openpoke/decidim-module-reporting-proposals/issues/78](https://link.getmailspring.com/link/6A205D54-42AB-4213-A035-F687F6699B51@getmailspring.com/0?redirect=https%3A%2F%2Fgithub.com%2Fopenpoke%2Fdecidim-module-reporting-proposals%2Fissues%2F78&recipient=bmFkamEudm9uYmFsbG1vb3NAc3RhZHRsdXplcm4uY2g%3D)

**Comment by carlobeltrame** [comment]
Habe das Modul geupdatet. @Naadjjaa bitte auf INT testen.

**Comment by Naadjjaa** [comment]
@carlobeltrame Es funktioniert nun alles wieder wie gewünscht.
Falls ihr sowieso einen PROD Release macht, bitte inkludieren.
@bsantschi Ansonsten ist es wohl zu kurzfristig dies heute noch zu machen?

**Comment by bsantschi** [comment]
Hoi Nadja, ja das ist zu kurzfristig, ich möchte PROD Releases jeweils vorher (idealerweise an einem Standup) mit dir, Lars und Anthony absprechen, so dass ihr (optional) auch noch mit den Endkunden abklären könnt, ob der Termin für alle in Ordnung ist.

**Comment by nicolefreypuzzle** [comment]
Hey @carlobeltrame 
Ist das wohl zwischenzeitlich auf PROD drin, seh ich das richtig? Oder benötigen wir hier noch Aktivitäten?

**Comment by nicolefreypuzzle** [comment]
@carlobeltrame kannst du mir zu meiner Frage vom 27.2. bitte noch eine Antwort geben? Merci!

**Comment by carlobeltrame** [comment]
> @carlobeltrame kannst du mir zu meiner Frage vom 27.2. bitte noch eine Antwort geben? Merci!

@nicolefreypuzzle ja, ist längst auf PROD, da wir seit das auf INT war schon mehrmals einPROD Deployment genacht haben.

**Comment by nicolefreypuzzle** [comment]
@NicolaNaf wie vermutet, ist das schon auf PROD. Hab den Task in die entsprechende Spalte verschoben. Nach erfolgreichem Test auf PROD von eurer Seite gerne ins Done schieben.

**Comment by larsUE** [comment]
Ich habe dies nun auf [INT getestet](https://test.dialogluzern.ch/admin/participatory_processes/asdfasdfas/components/720/manage/proposals) und es funktioniert.
